command_helper/get_keyword.py

```diff
@@ -164,5 +164,5 @@ def is_string(self, str_):
         if version_info.major > 2:
             status = isinstance(str_, str)
         else:
-            status = isinstance(str_, basestring)
+            status = isinstance(str_, str)

```

Comment:
With this change, the method looks like
```
    def is_string(self, str_):
        if version_info.major > 2:
            status = isinstance(str_, str)
        else:
            status = isinstance(str_, str)
        return status
```
which is definately wrong.

Reply:
Yeah, you totally right. I'll modify it.